src/main/java/com/rakovets/course/java/core/practice/looping_statements/Task05.java

```diff
@@ -34,6 +34,18 @@ public static void main(String[] args) {
     static String generateNumbersColumn(int numberRows, boolean isEnableHeaderRow) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return null;
+        String numberColumn = "";
+            if (isEnableHeaderRow) {
+                numberColumn += numberColumn + "\n";
+                numberRows -= 1;
+            }
+            for (int i =1; i <= numberRows; i++) {
+                if (i == numberRows) {
+                    numberColumn += i;
+                    break;
+                }
+                numberColumn += i + "\n";
+            }
+            return numberColumn;
```

Comment:
"Violation: There aren't correct indentations for inner code
Solution: Add correct indentations for inner code"